iocore/net/I_NetVConnection.h

```diff
@@ -644,6 +647,103 @@ class NetVConnection : public VConnection
   NetVConnection(const NetVConnection &) = delete;
   NetVConnection &operator=(const NetVConnection &) = delete;
 
+  enum ProxyProtocolVersion_t {
+    PROXY_VERSION_UNDEFINED,
+    PROXY_V1,
+    PROXY_V2,
+  };
+
+  enum ProxyProtocolData_t {
+    PROXY_PROTO_UNDEFINED,
+    PROXY_PROTO_SRC,
+    PROXY_PROTO_DST,
+  };
+
+  int
+  set_proxy_protocol_addr(ProxyProtocolData_t src_or_dst, ts::string_view &ip_port_str)
+  {
+    int ret = -1;
```

Comment:
Nitpicking, but hell, it's what I do. I much prefer an empty line between these local scope variables, and the code ;-).